libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -462,6 +458,14 @@ MQTTStatus_t MQTT_GetDisconnectPacketSize( size_t * pPacketSize );
  */
 MQTTStatus_t MQTT_SerializeDisconnect( const MQTTFixedBuffer_t * const pBuffer );
 
+/**
+ * @brief Get the size of an MQTT PINGREQ packet.
+ *
+ * @param[out] pPacketSize The size of the MQTT PINGREQ packet.
```

Comment:
Might as well just write `pPacketSize The size of the MQTT PINGREQ packet (2).` so we can quickly tell from the comment what it's supposed to return.

Reply:
Keeping the comment same as disconnect packet size.  We can update both  comments (and all other fixed size packet size ) later if we want.